components/analytics/true_up_review.scss

```diff
@@ -0,0 +1,77 @@
+.TrueUpReview__card {
+    width: 100%;
+    height: '463px';
+    border: 1px solid rgba(var(--sys-center-channel-color-rgb), 0.08);
+    background-color: var(--sys-center-channel-bg);
+    border-radius: 4px;
+    box-shadow: 0 2px 3px rgba(0, 0, 0, 0.08);
+    color: var(--sys-center-channel-color);
+}
+
+.TrueUpReview__cardHeader {
+    display: flex;
+    align-items: center;
+    padding: 28px 32px 24px 32px;
+    border-bottom: 1px solid rgba(var(--sys-center-channel-color-rgb), 0.08);
+}
+
+.TrueUpReview__cardHeaderText-top {
+    font-size: 16px;
+    font-weight: 600;
+    line-height: 24px;
+}
+
+.TrueUpReview__cardBody {
+    display: flex;
+    flex-direction: column;
+    padding: 20px;
+}
+
+.TrueUpReview__cardBody > * {
+    margin-bottom: 10px;
+}
+
+.TrueUpReview__cardBody > svg {
+    margin-left: 15px;
+}
+
+.dueDate :first-child {
+    color: #a3a4a8;
```

Comment:
Typically we try to stay with a palette color. It looks like the wireframe uses denim center channel text at 64% opacity. Since this component is rendered in system console where we try to not allow theme overrides, can we do color: rgba(var(--sys-center-channel-color-rgb), 0.64);` ?